common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/properties/SystemSubscriptionPropertiesDTO.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.cloud.notifications.models.dto.v1.endpoint.properties;
+
+import jakarta.validation.constraints.NotNull;
+
+import java.util.UUID;
+
+public final class SystemSubscriptionPropertiesDTO extends EndpointPropertiesDTO {
+    private UUID groupId;
+
+    @NotNull
+    private boolean ignorePreferences;
+
+    @NotNull
+    private boolean onlyAdmins;
```

Comment:
The deserialization error should be clearer (based on the annotation) if the fields are object booleans:
```suggestion
    @NotNull
    private Boolean ignorePreferences;

    @NotNull
    private Boolean onlyAdmins;
```

Reply:
Yes, changed to an object instead of using the primitives